web/src/containers/BudgetSummary.js

```diff
@@ -177,22 +172,13 @@ const BudgetSummary = ({ activities, data, years }) => (
           <th id="summary-budget-null2" />
           {[...years, 'total'].map(y => (
             <Fragment key={y}>
-              <th
-                className="right-align"
-                id={`summary-budget-fy-${y}-total`}
-              >
-                Total
+              <th className="right-align" id={`summary-budget-fy-${y}-total`}>
+                Medicaid total
```

Comment:
@jameshupp I lowercased "total" - it seemed like we were doing that in other places, where we only uppercase the first word.  Is that right?